components/sup/src/manager/service/config.rs

```diff
@@ -317,17 +317,17 @@ pub struct Cfg {
 }
 
 impl Cfg {
-    fn new(pkg: &Package) -> Result<Cfg> {
+    fn new<T: AsRef<Path>>(package: &Package, config_root: T) -> Result<Cfg> {
```

Comment:
Yeah, that's probably the right use of the `--config-from` that doesn't infect the code in a strange way. It'll be up to the Supervisor to strongly discourage using an override when not in development. This may also help us in testing